### PR TITLE
Add switch.template.

### PR DESCRIPTION
**Pull request by pavoni**
First cut - please check!

### CODE REVIEW

File: source/_components/switch.template.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+- **switches** array (*Required*): List of your switches.
+  - **friendly_name** (*Optional*): Name to use in the Frontend.
+  - **value_template** (*Required*): Defines a [template](/getting-started/templating/) to set the state of the switch.
+  - **turn_on** (*Required*): Defines an [action](/getting-started/automation/) to run when the switch is turned on.
+  - **turn_off** (*Required*): Defines an [action](/getting-started/automation/) to run when the switch is turned off.
+
+## {% linkable_title Examples %}
+
+In this section you find some real life examples of how to use this sensor.
```
Comment (balloob) [inline source/_components/switch.template.markdown]: use this *switch*

File: source/components/switch.template.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<script>document.location = '/components/switch.template/';</script>
```
Comment (balloob) [inline source/components/switch.template.html]: No need to add this file, it's only for backwards compatibility for components that existed before we migrated jekyll version

File: source/_components/switch.template.markdown [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+description: "Instructions how to integrate Template switches into Home Assistant."
+date: 2016-02-06 07:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+---
+
+The `template` platform creates switches that combine other components.
```
Comment (balloob) [inline source/_components/switch.template.markdown]: We could have some more explanation here how other components are actually combined.

**Comment by fabaff** [comment]
Sorry, I didn't check the PRs and added already something. Would be best to get rid of my commit, I assume.

**Comment by balloob** [comment]
I went ahead and copy pasted the content into the docs page. Fastest solution.

**Comment by pavoni** [comment]
Thanks guys.

**Pull request closed without merging**